### PR TITLE
FIX: Bug: when connected to custom Lndhub, create new ln wallet -> ba…

### PR DESCRIPTION
**Pull request by marcosrdz**
…ckup screen doesnt include this custom server

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ufdk9q1am94mvk7gjdzt2t46cr

[download apk](http://filestorage.bluewallet.io:1488/1610556705.apk)

File: screen/wallets/pleaseBackupLNDHub.js [hunk 1]
```diff
@@ -47,6 +47,10 @@ const PleaseBackupLNDHub = () => {
     };
   }, [handleBackButton]);
 
+  const secret =
```
Comment (Overtorment) [inline screen/wallets/pleaseBackupLNDHub.js:50]: nein. you just need to do `.getSecret()` it already includes code that appends custom server

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: screen/wallets/pleaseBackupLNDHub.js (hunk 1 above)
Comment (marcosrdz) [reply screen/wallets/pleaseBackupLNDHub.js]: addressed.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/qhph9u6p3fwpew8vnpav7rgq9g

[download apk](http://filestorage.bluewallet.io:1488/1610562007.apk)

**Review by Overtorment — APPROVED** [review]

**Review by ncoelho — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**